### DISCUSSION TITLE
Invoke-ArcGISConfiguration

### DISCUSSION THREAD

**Discussion by jajessup1981** [Esri/arcgis-powershell-dsc#522] [General]
Are we able to License/Configure a GIS Server Site with this command? I am looking through the arguments and only see [Install, Install/License, and Install/License/Configure]. I unfortunately already built out my components for Cloud Deployments, and would love to use ArcGIS-PS-DSC. Our team currently deploys Chef scripts for User Data configuration and would like to move away. 

Appreciate all you do for this Repo!

Looks like im going to roll w/ the Server Utilities to push config/licensing silently

**Comment by cameronkroeker** [comment]
Hi @jajessup1981,

The ArcGIS Module will search the Windows registry for the ArcGIS Server product code, if found then the installation is skipped for all modes (`-Mode Install`, `InstalLicense`, and `InstallLicenseConfigure`).

Thanks,
Cameron K.
